src/Parallel/centrality/betweenness.jl

```diff
@@ -2,9 +2,15 @@ betweenness_centrality(g::AbstractGraph, vs=vertices(g), distmx::AbstractMatrix=
 parallel == :distributed ? distr_betweenness_centrality(g, vs, distmx; normalize=normalize, endpoints=endpoints) : 
 threaded_betweenness_centrality(g, vs, distmx; normalize=normalize, endpoints=endpoints)
     
-betweenness_centrality(g::AbstractGraph, k::Integer, distmx::AbstractMatrix=weights(g); normalize=true, endpoints=false, parallel=:distributed) = 
-parallel == :distributed ? distr_betweenness_centrality(g, sample(vertices(g), k), distmx; normalize=normalize, endpoints=endpoints) :
-threaded_betweenness_centrality(g, sample(vertices(g), k), distmx; normalize=normalize, endpoints=endpoints)
+function betweenness_centrality(
+    g::AbstractGraph, k::Integer, distmx::AbstractMatrix=weights(g);
+    normalize=true, endpoints=false, parallel=:distributed,
+    rng::Union{Nothing, AbstractRNG}=nothing, seed::Union{Nothing, Integer}=-1
+)
+    samples = sample(vertices(g), k; rng=rng, seed=seed)
+    parallel == :distributed ? distr_betweenness_centrality(g, samples, distmx; normalize=normalize, endpoints=endpoints) :
```

Comment:
```suggestion
    return (parallel == :distributed) ? distr_betweenness_centrality(g, samples, distmx; normalize=normalize, endpoints=endpoints) :
```
We don't have any coding guidelines for Graphs.jl at the moment, but personally I perfer to explicetly use the `return` statement instead of just the last expression in a function. As this line is quite long, it might also be preferable to use an if-else statement instead of the tenary operator.

Reply:
> We don't have any coding guidelines for Graphs.jl at the moment

Exactly, and this really a pain.

> As this line is quite long, it might also be preferable to use an if-else statement instead of the tenary operator.

Can we please not discuss cosmetic / syntax changes, as this is always proven to be a waste of everybody's time ?
I've tried to stick to the conventions used in this repo, even if I don't agree with those, and especially because of inconsistencies between source files.
I've used the style / conventions that was used in previous `src/Parallel/...` files, e.g. on master:  https://github.com/JuliaGraphs/Graphs.jl/blob/a3fb98b6a2b07b392b5c690c057f1e5bfac25c9e/src/Parallel/centrality/betweenness.jl#L6.

Reply:
While I also don't like too waste much time on these, I think suggestions are always allowed - but maybe I should try to make it more clear in my writing that this really was just a suggestion. 

Anyway, as it is really minor and up to taste, lets ignore it for now.

Reply:
Thanks. This can always be done in a separate PR.